tools/scrapd-merger.py

```diff
@@ -57,8 +57,8 @@ def merge(old, new, update):
     :rtype: list(dict)
     """
 
-    old_dict = {entry['Case']: entry for entry in old}
-    new_dict = {entry['Case']: entry for entry in new}
+    old_dict = {entry["ID"]: entry for entry in old}
+    new_dict = {entry["ID"]: entry for entry in new}
```

Comment:
This will break the augmentations as they do not have this field. 

However you may be able to work around this using `rsplit('-', maxsplit=1)`. 

We should document that the format of the ID is `<case_number>-<deceased_number>`. The deceased number should be 0 based.